contracts/marketplace/CruzoMarket.sol

```diff
@@ -20,6 +20,8 @@ contract CruzoMarket is ERC1155Holder, Ownable {
 
     mapping(uint256 => Trade) private trades;
 
+    mapping(address => mapping(address => unit256)) private tradeCount;
```

Comment:
What about storing trades itself in a nested mapping instead of storing additional tradeCount mapping? Wouldn't it be a better design in our case?